easybuild/easyconfigs/v/VASP/VASP-6.3.2-nvofbf-2022.07.eb

```
@@ -0,0 +1,64 @@
+easyblock = 'MakeCp'
+
+name = 'VASP'
+version = '6.3.2'
+
+homepage = 'https://www.vasp.at'
+description = """The Vienna Ab initio Simulation Package (VASP) is a computer program for atomic scale
+materials modelling, e.g. electronic structure calculations and quantum-mechanical molecular dynamics,
+from first principles."""
+
+toolchain = {'name': 'nvofbf', 'version': '2022.07'}
+toolchainopts = {'openmp': True}
+
+download_instructions = """Vasp is proprietary software, see http://www.vasp.at/index.php/faqs
+ on how to get access to the code."""
+
+sources = ['%(namelower)s.%(version)s.tgz']
+checksums = ['f7595221b0f9236a324ea8afe170637a578cdd5a837cc7679e7f7812f6edf25a']
+
+prebuildopts = 'cp arch/makefile.include.nvhpc_omp_acc ./makefile.include && '
+
+# Makefile uses LIBS as a list of folders
+prebuildopts += 'unset LIBS && '
+
+local_targets = ['std', 'gam', 'ncl']
+
+buildopts = '%s ' % ' '.join(local_targets)
+buildopts += 'FFTW_ROOT="${EBROOTFFTW}" '
+
```

```suggestion
buildopts += 'DEPS=1 '  # required for VASP parallel builds
```
If you add that then you can remove the `parallel = 1` and the build successfully completes in parallel.

And thanks, I had need of this.